controllers/index.js

```diff
@@ -4,9 +4,11 @@ const homeRoutes = require('./home-routes.js');
 const dashboardRoutes = require('./dashboard-routes.js');
 const popularRoutes = require('./popular-routes.js');
 
+
```

Comment:
I think this index file either requires a user route reference, or the home-routes reference needs to be to the api user routes? maybe it works and I'm wrong

Reply:
I added the /users to the home-routes.js file, I think that was working